_common/GFMAlerts.php

```diff
@@ -56,42 +52,40 @@ protected function blockQuote($line)
                     array(
                         'name' => 'p',
                         'attributes' => array('class' => "markdown-alert-title title-$alertType"),
-                        'rawHtml' => $iconType
+                        'rawHtml' => $alertTitleHtml
                     ),
                 )
             )
         );
-        // dd($block);
         return $block;
     }
 
     protected function blockQuoteContinue($line, array $block) {
+        // If the block didn't start as an alert, fall through to blockquote handler
+        if(!isset($block['isAlert'])) {
+            return parent::blockQuoteContinue($line, $block);
+        }
+
         if(!preg_match('/^\>\s(.*)/', $line['text'], $matches)) {
-            return null;
+            return;
```

Comment:
It isn't clear to me if we should be `return null` or just `return`, because both seem to be used in Parsedown. However, `blockQuoteContinue` parent handler just does a `return` for these kinds of cases, so copied that